examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -1321,6 +1321,11 @@ impl FrontendNode {
             new_channel
```

Comment:
@daviddrysdale  After my change this line panics: `Step #8 - "run_tests": 2020-04-30 09:44:02,369 ERROR [sdk/rust/oak/src/lib.rs:424] panic occurred in file 'examples/abitest/module_0/rust/src/lib.rs' at line 1321: could not send request over channel: OakStatus(ErrChannelClosed)`, and then the call to `wait_on_channels` blocks for ever.  Do you know why this happens?

Reply:
I'm not completely sure, but my guess would be that the orphaned-channel logic (possibly including the early-return from `wait_on_channels`) is being triggered for a channel that has no writers but which still has a pending message.  IIRC a channel should only be considered orphan if it has no writers *and* no pending messages.